include/dqm4hep/AnalysisHelper.h

```diff
@@ -0,0 +1,65 @@
+/// \file AnalysislHelper.h
```

Comment:
This file should not be there. It's a file that is installed after `make install`